integration/tests/cook/test_basic.py

```diff
@@ -27,12 +27,18 @@ def get_job(self, job_uuid):
         """Loads a job by UUID using GET /rawscheduler"""
         return util.query_jobs(self.cook_url, job=[job_uuid]).json()[0]
 
+    def is_using_cook_executor(self):
+        return util.get_in(self.settings(), 'executor', 'command') is not None
```

Comment:
The cook executor is triggered only if the executor command is available.

Checks at:
https://github.com/twosigma/Cook/pull/308/files#diff-74ded9cfa2ff7c2832fa8c4b22ea40c6R340
https://github.com/twosigma/Cook/pull/308/files#diff-04ee7917c0d3616697cecca6d5c4c071R44